jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/nodes/CloudFormationExplorerNodeProvider.kt

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.explorer.nodes
+
+import com.intellij.openapi.project.Project
+import software.amazon.awssdk.services.cloudformation.CloudFormationClient
+import software.aws.toolkits.jetbrains.services.cloudformation.CloudFormationServiceNode
+import software.aws.toolkits.resources.message
+
+class CloudFormationExplorerNodeProvider : AwsExplorerServiceNode {
```

Comment:
Is this really a "node provider" ? its just a straight-up implementation of a root service node right?

Reply:
Correct.  Names have been changed in most recent commit.